src/planner/operator/mod.rs

```diff
@@ -27,6 +27,4 @@ pub enum Operator {
     Scan(ScanOperator),
     Sort(SortOperator),
     Limit(LimitOperator),
-    #[allow(dead_code)]
-    CreateTable(CreateTableOperator),
```

Comment:
insert and create table are not belong to operator ?

Reply:
Because the current `insert/create_table` is a single level, it will directly `build_create_table_logic_plan`, for now it is not necessary to add these two types in the operator